clients/include/utility.h

```diff
@@ -685,12 +685,11 @@ inline void rocblas_packInt8(host_vector<T>& A, rocblas_int M, rocblas_int N, ro
     }
 
     host_vector<T> temp(A);
-
```

Comment:
These changes won't help because inputs are still only rocblas_int?

Reply:
Nevermind - the index gets promoted to size_t as a result.

Reply:
just switch back to the original version if we aren't multi-threading this anyways.